api/RTCPeerConnection.json

```
@@ -2715,7 +2715,7 @@
               "version_added": false
```

```suggestion
              "version_added": "14.1"
```

I can confirm support using https://mdn-bcd-collector.appspot.com/tests/api/RTCPeerConnection/onicecandidateerror in Safari 14.1 and iOS 14.5.

It was added in https://trac.webkit.org/changeset/270101/webkit which is so recent that I'm not going to check further back.